sensor/NetworkStatsSensor.js

```diff
@@ -0,0 +1,174 @@
+/*    Copyright 2016 Firewalla LLC
+ *
+ *    This program is free software: you can redistribute it and/or  modify
+ *    it under the terms of the GNU Affero General Public License, version 3,
+ *    as published by the Free Software Foundation.
+ *
+ *    This program is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+'use strict';
+
+const log = require('../net2/logger.js')(__filename);
+
+const Sensor = require('./Sensor.js').Sensor;
+
+const fc = require('../net2/config.js');
+
+const FEATURE_NETWORK_STATS = "network_stats";
+const FEATURE_LINK_STATS = "link_stats";
+
+const rclient = require('../util/redis_manager.js').getRedisClient();
+
+const Ping = require('../extension/ping/Ping.js');
```

Comment:
2019-10-11 11:10:17 ERROR SensorLoader: Failed to load sensor: NetworkStatsSensor: Error: Cannot find module '../extension/ping/Ping.js'